charts/exivity/templates/pigeon/deployment.yaml

```diff
@@ -19,9 +19,21 @@ spec:
         checksum/{{- include "exivity.fullname" $ -}}-config-pigeon: {{ include (print $.Template.BasePath "/pigeon/configmap.yaml") . | sha256sum }}
     spec:
       volumes:
+        - name: projected
+          projected:
+            sources:
+              - configMap:
+                  name: {{ include "exivity.fullname" $ -}}-config-shared
+              - configMap:
+                  name: {{ include "exivity.fullname" $ -}}-licence-pub
+              - secret:
+                  name: {{ include "exivity.fullname" $ -}}-licence-key
         - name: config
           configMap:
             name: {{ include "exivity.fullname" $ -}}-config-pigeon
```

Comment:
The config is now no longer being used. Is that correct?

Reply:
fair point @Timmy1e , and that is probably incorrect, because my POD is now broken :-) need a bit more time to sort this out